lib/std/comptime_string_map.zig

```diff
@@ -9,26 +9,33 @@ const mem = std.mem;
 /// You can pass `struct { []const u8 }` (only keys) tuples if `V` is `void`.
 pub fn ComptimeStringMap(comptime V: type, comptime kvs_list: anytype) type {
     const precomputed = comptime blk: {
-        @setEvalBranchQuota(2000);
+        @setEvalBranchQuota(1500);
         const KV = struct {
             key: []const u8,
             value: V,
         };
         var sorted_kvs: [kvs_list.len]KV = undefined;
-        const lenAsc = (struct {
-            fn lenAsc(context: void, a: KV, b: KV) bool {
-                _ = context;
-                return a.key.len < b.key.len;
-            }
-        }).lenAsc;
         for (kvs_list, 0..) |kv, i| {
             if (V != void) {
                 sorted_kvs[i] = .{ .key = kv.@"0", .value = kv.@"1" };
             } else {
                 sorted_kvs[i] = .{ .key = kv.@"0", .value = {} };
             }
         }
-        mem.sort(KV, &sorted_kvs, {}, lenAsc);
+
+        const SortContext = struct {
+            kvs: []KV,
+
+            pub fn lessThan(comptime ctx: @This(), comptime a: usize, comptime b: usize) bool {
```

Comment:
Why make these comptime?

```suggestion
            pub fn lessThan(ctx: @This(), a: usize, b: usize) bool {
```

While you're at it, `comptime` can be removed from `comptime blk: {` above.